migration/migration_blackbox_test.go

```diff
@@ -209,6 +211,26 @@ func testMigration50(t *testing.T) {
 	assert.Nil(t, runSQLscript(sqlDB, "050-users-add-column-company.sql"))
 }
```

Comment:
Add these missing migrations:

```
func testMigration51(t *testing.T) {
    migrationToVersion(sqlDB, migrations[:(initialMigratedVersion+7)], (initialMigratedVersion + 7))

    assert.True(t, dialect.HasIndex("work_item_link_types", "work_item_link_types_name_idx"))
}

func testMigration52(t *testing.T) {
    migrationToVersion(sqlDB, migrations[:(initialMigratedVersion+8)], (initialMigratedVersion + 8))

    assert.True(t, dialect.HasIndex("spaces", "spaces_name_idx"))
}
```

I will have a look at why the migration isn't done until the specified version.